src/utils/wallet.js

```diff
@@ -30,7 +30,8 @@ export const WALLET_CREATE_NEW_ACCOUNT_FLOW_URLS = ['create', 'set-recovery', 's
 export const WALLET_LOGIN_URL = 'login'
 export const WALLET_SIGN_URL = 'sign'
 export const WALLET_RECOVER_ACCOUNT_URL = 'recover-account'
-export const ACCOUNT_HELPER_URL = process.env.REACT_APP_ACCOUNT_HELPER_URL || 'https://near-contract-helper.onrender.com'
+// export const ACCOUNT_HELPER_URL = process.env.REACT_APP_ACCOUNT_HELPER_URL || 'https://near-contract-helper.onrender.com'
```

Comment:
@marcinbodnar this seems like accidental debug leftover?

Reply:
I disabled CH in the last commit so you can test how it's working.